src/utils/image-processor.js

```diff
@@ -297,6 +297,100 @@ const getExif = function (file) {
   )
 }
 
+const getIccProfile = function (file) {
+  let iccProfile = null
+  const op = readJpegChunks(file)
+  op.progress(function (pos, l, marker, view) {
+    if (marker === 0xe2) {
+      // TODO: what kind of checks should be here?
```

Comment:
From `PIL/JpegImagePlugin.py`:
`if marker == 0xFFE2 and s[:12] == b"ICC_PROFILE\0":`

Reply:
Done